rootfs/usr/local/bin/generate-kubecfg

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -eo pipefail
+
+KUBECONFIG="${KUBECONFIG:-$HOME/.kube/config}"
+KUBECONFIG_DIR="$(dirname "$KUBECONFIG")"
+EXTRA_ARGS=${@:1}
+GOMPLATE_EXTRA_ARGS=${EXTRA_ARGS:-$GOMPLATE_EXTRA_ARGS}
+
+mkdir -p ${KUBECONFIG_DIR}
+
+function ensure_env_vars() {
+	export KOPS_API_SERVER="${KOPS_API_SERVER:-api.$KOPS_CLUSTER_NAME}"
```

Comment:
Unsure about `ensure_env_vars` assigning and asserting. We do more assigning above, so it feels like unnecessary sprawl